proteus/Domain.py

```diff
@@ -17,6 +19,17 @@ def __init__(self,nd,name="defaultDomain",units="m"):
         self.units=units
         self.x=[]#minx,miny,minz
         self.L=[]#bounding box when self.x is origin
+        self.g=g
+        # list of shape instances automatically attached to domain
+        self.shape_list = []
+        # list of auxiliaryVariables automatically attached to domaim
```

Comment:
If preferable, we could create a new class, such as `World()`, where we can attach variables like `g` and the `shape_list`, and call the domain functions from there instead. For now, the `shape_list` is necessary there for me because there could be several rigid bodies or obstacles within a tank, and it is easier to set them up separately rather than pass the full set of vertices, segments, etc.